_posts/2025-08-13-omero-web-5.29.2.md

```diff
@@ -0,0 +1,34 @@
+---
+layout: post
+title: Release of OMERO.web 5.29.2
+intro-blurb: The OME team is pleased to announce the release of OMERO.web 5.29.2
+---
+
+Today we are releasing OMERO.web 5.29.2, which includes 
+a fix for a security vulnerability, see 
+[CVE-2025-54791](https://github.com/ome/omero-web/security/advisories/GHSA-gpmg-4x4g-mr5r)
```

Comment:
Nitpicky, wondering if this should link back to the security advisory within this website. 
Since the content is replicated, it's not that critical.

Reply:
Previous (omero-web 5.26.0) announcement links to GitHub. Just confirming what @knabar said really, no opinion as not sure what is ``right`` here.